Dockerfile

```diff
@@ -15,13 +22,15 @@ RUN go mod download
 COPY cmd/main.go cmd/main.go
 COPY api/ api/
 COPY internal/ internal/
+COPY version/ version/
+COPY pkg/ pkg/
```

Comment:
This repo was created using the newer layout(v4) that uses an "internal" package for all private application code, anything in here isn't exposed by the go module see https://github.com/golang-standards/project-layout/tree/master/internal and https://go.dev/doc/go1.4#internalpackages

Everything you are doing here should be in the internal package.